### ISSUE TITLE
Dynamic REDCap Form Selection

### ISSUE THREAD

**Issue by PatrickDavisUW** [emerge-ehri/r4-hackathon#10]
Adding a way to configure what instruments/fields are pulled dynamically could allow for more flexible code as we individual sites could pull only the relevant fields

**Comment by rrfreimuth** [comment]
Agreed! We could create a generic client to pull data from R4, using a configurable list of fields (and authentication settings). I hope we can get a start on this during the hackathon today.
